OpenOversight/app/models.py

```diff
@@ -129,6 +182,10 @@ class Officer(BaseModel):
     unique_internal_identifier = db.Column(
         db.String(50), index=True, unique=True, nullable=True
     )
+    date_created = db.Column(db.DateTime, default=func.now())
+    date_updated = db.Column(
+        db.DateTime, default=func.now(), onupdate=func.now(), index=True
+    )
```

Comment:
This might make bulk updates really slow since we'd be updating lots of values in this column

Reply:
I think even at scale that shouldn't be too impactful, the rows are getting updated anyway and in Postgres that means pulling out the whole row at once. I think we should be okay!